### PR TITLE
Cmake fix for cpp-httplib

### PR DESCRIPTION
**Pull request by nithinp7**
Possibly fixes: #251. 

Tells `cpp-httplib` not to look for zlib or openssl. Quoting @baothientran for what the issue may have been:
>hm good chance is your dev environment have zlib and openssl installed and httplib link against them when building. But those libraries are not copied in to unreal third party and it didn't know how to link against those symbol 

I'm not going to pretend I understand cmake or what exactly httplib, zlib, and openssl do, but this change finally solved a very painful issue I was stuck on for the last two days :). The issue does not happen in everybody's dev environment, so I think Bao's idea makes a lot of sense. 

@jtorresfabra @kring

### CODE REVIEW

**Comment by shehzan10** [comment]
@nithinp7 I had run into the OpenSSL one with Android. We added the same option to the toolchain file. This looks like the right fix to me

https://github.com/CesiumGS/cesium-unreal/blob/android-platform/extern/unreal-android-toolchain.cmake#L15.

**Comment by kring** [comment]
We're using cpp-httplib as a tiny web server we can run on localhost as part of the OAuth handshake with Cesium ion. Because we're only running on localhost, we don't need https support, so we don't need OpenSSL. We also don't particularly care whether the responses returned from our server are compressed, so we don't need zlib either.

If we ever start using cpp-httplib's HTTP _client_ support, both of these will be much more important. And we'll probably need to include zlib and openssl properly. This is pretty unlikely, though, considering that cpp-httplib requests are blocking.

Long story short, this looks good!
